ceremonies/src/tests.rs

```diff
@@ -788,20 +542,18 @@ fn claim_rewards_works() {
 		assert!(event_deposited::<TestRuntime>(
 			BalancesEvent::Issued(
 				cid,
-				account_id(&bob),
+				bob.clone(),
 				EncointerCeremonies::ceremony_reward().lossy_into()
 			)
 			.into()
 		));
 
 		assert!(event_deposited::<TestRuntime>(
-			Event::NoReward {
+			BalancesEvent::Issued(
 				cid,
-				cindex,
-				meetup_index: 1,
-				account: account_id(&ferdie),
-				reason: ExclusionReason::TooFewOutgoingAttestations,
-			}
+				charlie.clone(),
+				EncointerCeremonies::ceremony_reward().lossy_into()
```

Comment:
Charlie does now receive rewards because he supplies valid attestations. Before, he handed in claims with invalid signatures.